src/olympia/search/filters.py

```diff
@@ -913,9 +913,11 @@ def apply_search_query(self, search_query, qs, sort=None):
             ),
         ]
         ranking_bump_groups = amo.utils.sorted_groupby(
-            PROMOTED_GROUPS, lambda g: g.search_ranking_bump, reverse=True
+            PROMOTED_GROUP_CHOICES.ACTIVE.entries,
+            lambda g: getattr(g, 'search_ranking_bump', 0.0),
```

Comment:
I wonder if we should do this at indexing time as well. It would mean a reindex of affected add-ons if we ever change the value for a given group, but it's not that much worse than having to redeploy...

Reply:
How would that work in the query?  

(Not that it blocks us from adding it to the index in this patch, if you're confident it's possible)